### PR TITLE
Use googleapis common protos

### PR DESCRIPTION
**Pull request by dhermes**
~~Has #1317 as diffbase.~~

H/T to @tbetbetbe who alerted me to the existence of https://pypi.python.org/pypi/googleapis-common-protos. w00t!

### CODE REVIEW

**Comment by dhermes** [comment]
@tseaver PTAL I rebased after #1317

File: scripts/make_operations_grpc.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            grpc_only_lines.append(line)
+
+    return grpc_only_lines
+
+
+def main():
+    """Write gRPC-only lines to custom module."""
+    grpc_only_lines = get_pb2_grpc_only()
+    with open(GRPC_ONLY_FILE, 'wb') as file_obj:
+        file_obj.write(''.join(grpc_only_lines))
```
Comment (tseaver) [inline scripts/make_operations_grpc.py]: `file_obj.writelines(grpc_only_lines)` would probably be smoother.

**Comment by tseaver** [comment]
LGTM, even if you decide not to address the earlier `writelines` comment.

File: scripts/make_operations_grpc.py (hunk 1 above)
Comment (dhermes) [inline scripts/make_operations_grpc.py]: :+1: good call. Memory saver.

**Pull request merged**